frontend/src/components/NewBoardModal.css

```diff
@@ -0,0 +1,113 @@
+.newBoard-overlay2 {
+  display: flex;
+  position: fixed;
+  z-index: 1000;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(1, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+main {
+  background-color: white;
+  flex-grow: 1;
+}
+
+.newBoard-details h2 {
+  padding: 0px;
+  margin: 10px;
+}
+
+.newBoard-details p:hover {
```

Comment:
Here and below, better to use classes, not html tags, in case you introduce another element of the same type, but without these styles.